keyring/backends/SecretService.py

```diff
@@ -52,6 +52,9 @@ def get_password(self, service, username):
         items = collection.search_items(
             {"username": username, "service": service})
         for item in items:
+            if hasattr(item, 'unlock'):
+                if item.is_locked() and item.unlock()[0]:
```

Comment:
I was reviewing the keyring SecretService integration code and noticed this line.

`item.unlock()` [returns a boolean](https://secretstorage.readthedocs.io/en/latest/item.html#secretstorage.item.Item.unlock) and it has been so since the introduction of that method in mitya57/secretstorage@0f194a5f3cb1f1ca2c82049618e478365b07c25d.

@mathstuf Why did you use `[0]` here?

Reply:
As I got no response, I went ahead and fixed this in b08b6846e8705b8ddd2ee7aed61ab6822f0870be.

Reply:
Indeed, I missed that. Sorry for the delay; just got back from an extended vacation.